src/CqlBuilderPanel/functionsSection/functionBuilder/FunctionBuilder.tsx

```diff
@@ -23,11 +26,14 @@ export interface Funct {
 }
 
 export interface FunctionProps {
+  cqlBuilderLookupsTypes: CqlBuilderLookup;
   canEdit: boolean;
   handleApplyFunction: Function;
   handleFunctionEdit?: Function;
   funct?: Funct;
   onClose?: Function;
+  operation?: string;
+  cqlFunction?: any;
```

Comment:
Are these required for the future stories ?

Reply:
operation should be for edit capability. I think cqlFunction can be removed.